src/XMakeTasks/Al.cs

```diff
@@ -292,7 +292,7 @@ override protected string ToolName
         {
             get
             {
-                return NativeMethodsShared.IsWindows ? "AL.exe" : "al";
+                return "al.exe";
```

Comment:
I think we rely on `al` (i.e. the wrapper script that then calls into al.exe) on Mono/xbuild. Is that not the case here?